src/pages/tasks/NewTaskDetailsPage.js

```diff
@@ -80,13 +98,18 @@ const NewTaskPage = (props) => {
                         ref={(el) => (inputRef.current = el)}
                         inputID="taskTitle"
                         label={props.translate('newTaskPage.title')}
+                        defaultValue={props.task.title}
```

Comment:
Isn't defaultValue reduntant here? Can we remove it if so?

Reply:
This is correct. Will remove it.